src/Giraffe/FormatExpressions.fs

```diff
@@ -80,4 +80,57 @@ let tryMatchInput (format : PrintfFormat<_,_,_,_, 'T>) (input : string) (ignoreC
             :?> 'T
             |> Some
     with
-    | _ -> None
\ No newline at end of file
+    | _ -> None
+
+let matchDict = 
```

Comment:
Can we make this guy private perhaps?

Reply:
I've made private, given these are only run at startup won't impact perf constructing on each call, I can even change to a match statement which is probably better as then literals.